compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformTextInputMethodRequest.skiko.kt

```diff
@@ -47,6 +53,9 @@ actual interface PlatformTextInputMethodRequest {
     @ExperimentalComposeUiApi
     val editProcessor: EditProcessor?
 
+    @ExperimentalComposeUiApi
```

Comment:
Maybe we better clarify what exactly it is?

```suggestion
    /** Untransformed text TextFieldValue */
    @ExperimentalComposeUiApi
```

Reply:
Judging from the implementation of `TransformedTextFieldState.collectImeNotifications`, it's actually the (transformed) `visualText`.

Reply:
I'd like to document what it is, but I don't really understand - when is `NotifyImeListener` called? Do we actually need it?